modules/configuring-microshift-virtual-machine.adoc

```diff
@@ -0,0 +1,53 @@
+// 
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-microshift-virtual-machine"]
+= Configuring the MicroShift virtual machine
+:context: creating-microshift-development-environment
+
+Use the following procedure to configure a virtual machine for MicroShift 
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+
+.Procedure 
+
+. Run the following command to configure root privileges: 
++
+[source,terminal]
+----
+$ echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
+----
+
+. Run the following command to upgrade your system:
++
+[source,terminal]
+----
+$ sudo dnf update -y
+----
+
+. Run the following command to install basic dependencies: 
++
+[source,terminal]
+----
+$ sudo dnf install -y git cockpit make gcc selinux-policy-devel rpm-build bash-completion
+----
+
```

Comment:
```suggestion
```

@ggiguash do we really need cockpit, or is it just convenient? I never use it on my system. I'm fairly certain none of the other things are dependencies for just running MicroShift, so I think we can remove this step entirely unless we do need cockpit.